e2e_test/sink/kafka/append_only1.result

```diff
@@ -1,10 +1,10 @@
-{"id":1,"v_bigint":1872,"v_bytea":"AA==","v_double":23956.39329760601,"v_float":26261.416015625,"v_integer":1872,"v_smallint":31031,"v_timestamp":1681453634104,"v_varchar":"8DfUFencLe"}
-{"id":10,"v_bigint":20674,"v_bytea":"AA==","v_double":9042.404483827513,"v_float":19387.23828125,"v_integer":20674,"v_smallint":26951,"v_timestamp":1681404058888,"v_varchar":"0oVqRIHqkb"}
-{"id":2,"v_bigint":4598,"v_bytea":"AA==","v_double":31923.077305746086,"v_float":27031.224609375,"v_integer":4598,"v_smallint":22690,"v_timestamp":1681429444869,"v_varchar":"sIo1XXVeHZ"}
-{"id":3,"v_bigint":14894,"v_bytea":"AAAA","v_double":9742.475509566086,"v_float":2660.290283203125,"v_integer":5894,"v_smallint":5985,"v_timestamp":1681429011269,"v_varchar":"LVLAhd1pQv"}
-{"id":4,"v_bigint":24962,"v_bytea":"3q2+7w==","v_double":3119.719721891862,"v_float":21217.77734375,"v_integer":7406,"v_smallint":6306,"v_timestamp":1681434727993,"v_varchar":"ORjwy3oMNb"}
-{"id":5,"v_bigint":9253,"v_bytea":"/u2+7w==","v_double":17464.91553421121,"v_float":22749.5,"v_integer":9253,"v_smallint":22765,"v_timestamp":1681444642324,"v_varchar":"sSkKswxrYd"}
-{"id":6,"v_bigint":28842,"v_bytea":"yv66vg==","v_double":11210.458724794062,"v_float":5885.3681640625,"v_integer":10844,"v_smallint":4014,"v_timestamp":1681382522137,"v_varchar":"V4y71v4Gip"}
-{"id":7,"v_bigint":15914,"v_bytea":"ur7A3g==","v_double":10967.182297153104,"v_float":3946.743408203125,"v_integer":12652,"v_smallint":10324,"v_timestamp":1681447263083,"v_varchar":"YIVLnWxHyf"}
-{"id":8,"v_bigint":28641,"v_bytea":"3q26vg==","v_double":993.408963466774,"v_float":13652.0732421875,"v_integer":19036,"v_smallint":194,"v_timestamp":1681393929356,"v_varchar":"lv7Eq3g8hx"}
-{"id":9,"v_bigint":24837,"v_bytea":"3q26vg==","v_double":11615.276406159757,"v_float":20699.55859375,"v_integer":20090,"v_smallint":10028,"v_timestamp":1681389642487,"v_varchar":"nwRq4zejSQ"}
+{"id":1,"v_bigint":1872,"v_bytea":"AA==","v_date":1,"v_double":23956.39329760601,"v_float":26261.416015625,"v_integer":1872,"v_interval":"P0Y0M0DT0H0M0S","v_jsonb":"{}","v_smallint":31031,"v_time":1000,"v_timestamp":1681453634104,"v_timestamptz":"0001-01-01 00:00:00.123456","v_varchar":"8DfUFencLe"}
```

Comment:
Note to myself: this would become `"v_timestamptz":"0001-01-01 00:00:00.123456Z"` after migrating to new syntax #13066